Library/Formula/libodfgen.rb

```diff
@@ -1,5 +1,5 @@
 class Libodfgen < Formula
-  homepage "http://sourceforge.net/p/libwpd/wiki/libodfgen/"
+  homepage "https://sourceforge.net/p/libwpd/wiki/libodfgen/"
```

Comment:
I'm a little :neutral_face: on this. For users who aren't logged in it automatically redirects immediately to plaintext, there's no real SSL/TLS element for unregistered users.